kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/QueryServiceRestOnlyIntegrationTest.java

```diff
@@ -55,6 +55,6 @@ public void testFindTaskById() throws Exception {
         assertTrue(tasks.length > 0);
         WebTarget target = newRequest(RestURI.build(TestConfig.getKieServerHttpUrl(), QUERY_URI + "/" + TASK_GET_URI, Collections.singletonMap(TASK_INSTANCE_ID, tasks[0].getId())));
         assertTrue(target.request(MediaType.APPLICATION_JSON_TYPE).get(String.class).contains("null"));
-        assertFalse(target.request().header("content-type", MediaType.APPLICATION_JSON + ";fields=not_null,strict=true").get(String.class).contains("null"));
+        assertFalse(target.request().header("content-type", MediaType.APPLICATION_JSON + ";fields=not_null").get(String.class).contains("null"));
```

Comment:
Can you please try ";fields=not_null;strict=true"?
I guess I made a typo when originaly writting the test, at least according to https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.7
Probably WebLogic is more strict than the others applicatiion servers.

Reply:
Seems that `;` should be used rather than `,`